$DragonFly: src/secure/usr.bin/ssh/Attic/ssh_config.5.no_obj.patch,v 1.5 2007/09/08 05:01:18 pavalos Exp $
--- ../../../crypto/openssh-4/ssh_config.5	2006-08-04 18:34:51.000000000 -0700
+++ ssh_config.5	2006-11-20 09:48:54.000000000 -0700
@@ -165,7 +165,7 @@
 .Dq no ,
 the check will not be executed.
 The default is
-.Dq yes .
+.Dq no .
 .It Cm Cipher
 Specifies the cipher to use for encrypting the session
 in protocol version 1.
@@ -1035,6 +1035,11 @@
 .Sx VERIFYING HOST KEYS
 in
 .Xr ssh 1 .
+.It Cm VersionAddendum
+Specifies a string to append to the regular version string to identify
+OS- or site-specific modifications.
+The default is
+.Dq DragonFly-20070907 .
 .It Cm XAuthLocation
 Specifies the full pathname of the
 .Xr xauth 1
